### PR TITLE
Remove changelog

### PR DESCRIPTION
**Pull request by robertknight**
The client’s deploy process has some failure modes that only exist because the release process involves a commit to master to update `package.json` and `CHANGELOG.md`. As discussed in Slack, we're going to simplify things as follows:

1. Remove the changelog from the source tree. We'll just use GitHub release notes instead, as well as handwritten notes in the #changelog Slack channel.
2. Replace the version in `package.json` with a dummy version which is only set when the release is actually created

This PR implements the first step of that plan by removing the changelog file and references to it. The GitHub release script is updated to generate the release notes directly instead of reading from the changelog.

### CODE REVIEW

**Comment by robertknight** [comment]
This PR will need updating because the API for passing OAuth access tokens to Octokit changed: https://github.com/octokit/rest.js#authentication

File: scripts/gulp/live-reload-server.js [hunk 1]
```diff
@@ -15,8 +15,8 @@ function licenseText() {
   return fs.readFileSync('./LICENSE', 'utf-8');
 }
 
-function changelogText() {
-  return fs.readFileSync('./CHANGELOG.md', 'utf-8');
+function codeOfConductText() {
```
Comment (lyzadanger) [inline scripts/gulp/live-reload-server.js]: Can you help me understand how these changes fit in with this set of changes?

File: scripts/create-github-release.js [hunk 2]
```diff
@@ -9,59 +9,43 @@
  * `v<VERSION>` where <VERSION> is the `version` field in package.json.
  */
 
-const fs = require('fs');
-const request = require('request');
+const Octokit = require('@octokit/rest');
```
[the remark this answers is not in the document]
Comment (robertknight) [reply scripts/create-github-release.js]: I thought so too. They did break their API in a non-major release recently though. Grr...

File: scripts/create-github-release.js [hunk 3]
```diff
@@ -9,59 +9,43 @@
...
-  tag_name: `v${pkg.version}`,
-  name: `v${pkg.version}`,
-  body: extractReleaseNotes(changelog, pkg.version),
-  draft: false,
-  prerelease: true,
-};
+  await octokit.repos.createRelease({
+    body: changes,
+    draft: false,
+    name: `v${pkg.version}`,
```
[the remark this answers is not in the document]
Comment (robertknight) [reply scripts/create-github-release.js]: Yes, indeed.

File: scripts/gulp/live-reload-server.js (hunk 1 above)
Comment (robertknight) [reply scripts/gulp/live-reload-server.js]: The client's dev server used CHANGELOG.md as a dummy document. That's being removed so it needs to render something else instead.

**Pull request merged**